src/redux/migrations.ts

```diff
@@ -793,15 +793,28 @@ export const migrations = {
     },
   }),
   72: (state: any) => ({
+    ...state,
+    app: {
+      ...state.app,
+      showGuidedOnboardingCopy: REMOTE_CONFIG_VALUES_DEFAULTS.showGuidedOnboardingCopy,
+    },
+  }),
+  73: (state: any) => ({
     ...state,
     app: {
       ...state.app,
       shouldShowRecoveryPhraseInSettings:
         REMOTE_CONFIG_VALUES_DEFAULTS.shouldShowRecoveryPhraseInSettings,
     },
   }),
-  73: (state: any) => state,
   74: (state: any) => ({
+    ...state,
+    app: {
+      ...state.app,
+      inviteMethod: REMOTE_CONFIG_VALUES_DEFAULTS.inviteMethod,
```

Comment:
What does this inviteMethod have to do with your PR?

Reply:
> What does this inviteMethod have to do with your PR?

It does not; I must have gotten confused while merging changes from main.